lib/spack/env/cc

```diff
@@ -175,6 +182,37 @@ fi
 input_command="$@"
 args=("$@")
 
+# Prepend cppflags, cflags, cxxflags, fcflags, fflags, and ldflags
+
+# Add cppflags
+case "$mode" in
+    cppas|cc|ccld)
+        args=(${SPACK_CPPFLAGS[@]} "${args[@]}") ;;
+esac
+
+# Add compile flags.
+case "$mode" in
+    cc|ccld)
+    # Add c, cxx, fc, and f flags
+        case $lang_flags in
+            C)
+                args=(${SPACK_CFLAGS[@]} "${args[@]}") ;;
+            CXX)
+                args=(${SPACK_CXXFLAGS[@]} "${args[@]}") ;;
+            FC)
```

Comment:
Note that I added `FCFLAGS` for `F90`, to be compatible with `autotools` and other build systems.